tests/factories/file.js

```diff
@@ -3,25 +3,33 @@ import faker from 'faker';
 
 FactoryGuy.define('file', {
     default: {
-        name: () => faker.lorem.word() + '.txt',
+        name: () => faker.lorem.words(1) + '.txt',
```

Comment:
I know that `lorem.word` is available in newer versions of faker. Am I correct in assuming this is a workaround for old version issues?